clients/python/example_check.py

```diff
@@ -0,0 +1,20 @@
+from .kh_client import *
+import os
+
+fail = os.getenv("fail", False)
```

Comment:
Environment variables are typically uppercase, could you uppercase this env var?